shapely/geometry/base.py

```diff
@@ -197,6 +197,22 @@ def __sub__(self, other):
     def __xor__(self, other):
         return self.symmetric_difference(other)
 
+    def __eq__(self, other):
+        if not isinstance(other, BaseGeometry):
+            return NotImplemented
+        # equal_nan=False is the default, but not yet available for older numpy
```

Comment:
Maybe put a note that it was introduced in version 1.19.0, which I imagine will be the minimum version in a few years.